### DISCUSSION TITLE
Jogging not working

### DISCUSSION THREAD

**Discussion by grayjamz** [arkypita/LaserGRBL#835] [Q&A]
Hello,
I'm a total noob to Laser GRBL 
I installed laser grbl 3.0.24 and hooked it up to my arduino set up.
If I upload a Gcode program it works and moves the stepping motors without any issue.
But If I want to move the axis manual using the jogging control which is not grayed out it does not do anything.
What am I missing,
I would appreciate a nudge in the right direction 
Thanks in advance
Regards

**Comment by StuartB4** [comment]
Which version of GRBL do you have? 

![676767](https://user-images.githubusercontent.com/16980026/72031736-30dc3b00-3285-11ea-9e41-a3a41c58c181.jpg)

**Comment by grayjamz** [comment]
It says Grbl 1.1z

**Comment by arkypita** [comment]
1.1z is an unofficial, unsupported, non standard version that I don't know who has created it.

The owner/developer of grbl also does not know where this version of the firmware come from.

https://github.com/gnea/grbl/issues/775#issuecomment-567160226

**Comment by arkypita** [comment]
Can you tell me more about your hardware? Where did you buy it? What control board does it mount?

**Comment by StuartB4** [comment]
Maybe a Bachin machine. I've seen a few of their machines with 1.1z.

**Comment by grayjamz** [comment]
As I said I'm a total noob.
I bought the hardware on ebay because a channel of the previous  hardware that ran on mach3  stopped responding and I wanted to go USB instead of serial.
I would not be able to tell you more about the hardware. Below is a link to the one I bought on ebay.

https://www.ebay.com/itm/GRBL-Laser-Controller-Board-USB-3-Axis-Stepper-Motor-CNC-Driver-Controller-Board/142913976919?hash=item2146563a57:g:KjIAAOSwhh9byXrS

**Comment by grayjamz** [comment]
would I be able to flash it to the official GRBL version?

**Comment by arkypita** [comment]
Yes it is possible, but if the producer of that hardware has made some specific customization in this 1.1z firmware there is a few possibility that it doesn't work and you cannot go back

**Comment by arkypita** [comment]
Safe way is to buy a fresh new arduino nano (the blue board on the main board), detach the old board and flash the official version on the new one, so you have the original firmware on the original board.

**Comment by arkypita** [comment]
You are not the only one with this issue:

https://github.com/arkypita/LaserGRBL/issues/543

https://github.com/gnea/grbl/issues/659

**Comment by grayjamz** [comment]
arkypita thank you very much.
Are there fake arduino nanos or can I buy them anywhere without having to worry with counterfeit ones?
If there are counterfeit ones can you recommend where I can buy them?

**Comment by arkypita** [comment]
99% of arduino nano you can buy are Chinese clone but they work well. Original arduino comes only from arduino site shop and from some authorized reseller.

**Comment by arkypita** [comment]
However if you feel lucky you can try flashing directly your board. Mine was only a warning 😉

**Comment by grayjamz** [comment]
That is the plan exactly. If it does not work I order a new one. 
Thanks allot again.
